pallets/teerex/ias-verify/src/lib.rs

```diff
@@ -133,6 +139,70 @@ pub struct SgxReport {
 	pub build_mode: SgxBuildMode,
 }
 
+#[derive(Encode, Decode, Clone, TypeInfo, sp_core::RuntimeDebug, Default)]
+pub struct SgxQuoteInputs {
+	pub spid: [u8; 16],
+	pub nonce: [u8; 16],
+
+	// the revocation list
+	pub sig_rl: Vec<u8>,
+}
+
+#[derive(Encode, Decode, Clone, TypeInfo, Default, sp_core::RuntimeDebug)]
+pub struct SgxQuoteAdd {
+	pub quote_inputs: SgxQuoteInputs,
+}
+
+#[derive(Encode, Decode, Clone, TypeInfo, sp_core::RuntimeDebug)]
+pub struct SgxTargetInfo {
+	// target info
+	pub mr_enclave: [u8; 32],
+	pub attributes: SGXAttributes,
+	pub reserved1: [u8; SGX_REPORT_BODY_RESERVED1_BYTES],
+	pub config_svn: u16,
+	pub misc_select: [u8; 4],
+	pub reserved2: [u8; SGX_REPORT_BODY_RESERVED2_BYTES],
+	pub config_id: [u8; 64],
+	pub reserved3: [u8; SGX_REPORT_BODY_RESERVED3_BYTES],
```

Comment:
we can sort the items here to make it graceful, like 

```
	
{
	pub attributes: SGXAttributes,
        pub config_svn: u16,
        pub config_id: [u8; 64],
	pub misc_select: [u8; 4],
        pub mr_enclave: [u8; 32],	
        pub reserved1: [u8; SGX_REPORT_BODY_RESERVED1_BYTES],
        pub reserved2: [u8; SGX_REPORT_BODY_RESERVED2_BYTES],
        pub reserved3: [u8; SGX_REPORT_BODY_RESERVED3_BYTES],
}
```

Reply:
i'ts ok to list the `SgxTargetInfo`  fields by order of the `sgx_target_info_t` structure.